inc/admin.php

```diff
@@ -248,3 +258,20 @@ function get_error_messages() {
 
 	return $messages;
 }
+
+/**
+ * Check whether there is an existing ads.txt file that will override the plugin output.
+ *
+ * @return bool Value of whether ads.txt already exists
+ */
+function ads_txt_exists() {
+	// No-op on WordPress.com
+	if ( defined( 'WPCOM_IS_VIP_ENV' ) && WPCOM_IS_VIP_ENV ) {
+		return;
+	}
+
+	$ads_txt_path = get_site_url() . '/ads.txt';
```

Comment:
Could this fail due to CDN type caching? Probably worth adding a query vary with the current time in milliseconds to help break caching for rechecks after removing.

Reply:
Removed due to refactoring as a JS-only check, but will add the query string in the JS implementation.

Reply:
https://github.com/10up/ads-txt/pull/19/commits/aa7e321fdb673a396bb6ce1fc92d9330476c60e5